fulp_modules/features/exclusive_fauna/exiled_king_basic.dm

```diff
@@ -0,0 +1,582 @@
+#define BB_KING_PORTAL "BB_king_portal"
+#define BB_KING_TENTACLE "BB_king_tentacle"
+#define BB_KING_SURROUND_TENTACLE "BB_king_surround_tentacle"
+#define BB_CHARGE_ABILITY "BB_charge_ability"
+#define BOSS_MEDAL_EXILED "Exiled Killer"
+#define EXILED_KING_SCORE "EXiled Killed"
+
+/mob/living/basic/exiled_king
+	name = "Exiled King"
+	desc = "and what army?"
+	health = 2500
+	maxHealth = 2500
+	attack_verb_continuous = "slaps"
+	light_range = 3
+	attack_verb_simple = "slap"
+	attack_sound = 'sound/magic/demon_attack1.ogg'
+	icon_state = "cthulu"
+	icon_living = "cthulu"
+	icon_dead = ""
+	health_doll_icon = "cthulu"
+	friendly_verb_continuous = "stares down"
+	friendly_verb_simple = "stare down"
+	combat_mode = TRUE
+	faction = list(FACTION_MINING, FACTION_BOSS)
+	unsuitable_atmos_damage = 0
+	minimum_survivable_temperature = 0
+	maximum_survivable_temperature = INFINITY
+	icon = 'fulp_modules/features/exclusive_fauna/icons/96x96.dmi'
+	speak_emote = list("gurgles")
+	armour_penetration = 60
+	melee_damage_lower = 60
+	melee_damage_upper = 60
+	speed = 5
+	mob_size = MOB_SIZE_LARGE
+	pixel_x = -32
+	base_pixel_x = -32
+	maptext_height = 96
+	faction = list(FACTION_MINING, FACTION_BOSS)
+	maptext_width = 96
+	basic_mob_flags = DEL_ON_DEATH
+	death_message = "sinks into a pool of blood, fleeing the battle. You've won, for now... "
+	death_sound = 'sound/magic/enter_blood.ogg'
+	ai_controller = /datum/ai_controller/basic_controller/kraken
+	///achievement for killing
+	var/achievement_type = /datum/award/achievement/boss/king_slayer
+	///score of killings
+	var/score_achievement_type = /datum/award/score/king_score
+
+
+/mob/living/basic/exiled_king/Initialize(mapload)
+	. = ..()
+	AddComponent(/datum/component/gps, "Sunken Signal")
+	add_traits(list(TRAIT_LAVA_IMMUNE, TRAIT_ASHSTORM_IMMUNE), INNATE_TRAIT)
+	AddElement(/datum/element/simple_flying)
+	AddElement(/datum/element/crusher_loot, /obj/item/crusher_trophy/bileworm_spewlet, 15)
```

Comment:
no unique crusher trophy?

Reply:
ill think up of a unique crusher before i undraft this